### DISCUSSION TITLE
What cloud provider can you use to host Postal? Port 25 is blocked by all providers

### DISCUSSION THREAD

**Discussion by collimarco** [postalserver/postal#2579] [Help with using Postal]
Hello,

I am really interested in Postal as a way to be independent from major email providers (like Sendgrid, AWS SES, etc.).

However it seems that all cloud providers / infrastructure providers (DigitalOcean, AWS, OVH, etc.) are blocking the port 25 of the VMs / servers.

What provider can you use to run Postal? Do you have a list? Any recommendation would be greatly appreciated.

**Comment by sinnbeck** [comment]
I personally use hetzner. Yes port 25 etc is closed, but I was able to get it open by asking. I don't know if you need to be a customer for some time for them to open it.

If you already have a server somewhere try asking them

**Comment by willpower232** [comment]
> If you already have a server somewhere try asking them

This is the best answer. I don't know if there is such a list on the internet but there isn't one maintained here I'm afraid.
